docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,818 @@
+
+<p class="blog-description">1497 out of 1589 answered.</p>
+
+This year we saw a considerable bump in Next.js usage, which is not surprising due to its increasing popularity and the advancements it gained in 2021.
+
+### What styling solution are you using?
+
+<img src="/static/blog/2021-survey/22.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="MUI Core v4 (JSS): 45%; Styled components: 37.9%; Emotion: 30.2%; SASS: 20.8%; CSS modules: 18.9%; Vanilla CSS: 17.6%; Tailwind CSS: 9.1%; Stitches: 0.4%; Other: 1.2%" />
+
+<p class="blog-description">1492 out of 1589 answered.</p>
+
+As the MUI Core v5 release is relatively recent, we expect to see many developers still using JSS, and there will probably always be some of you who will prefer it.
+
+But the growth of Emotion and styled-components here is noteworthy, because they are what our new styling solution is based on.
+This year we'll be focusing on expanding and refining [MUI System](https://mui.com/system/basics/) to further improve upon this new styling solution.
+
+### Did you recently migrate to MUI Core v5?
+
+<img src="/static/blog/2021-survey/23.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Yes: 63.3%; No: 36.7%" />
+
+<p class="blog-description">1546 out of 1589 answered.</p>
+
+Though many of you who answered the survey have already migrated, we still have many more to convince to come along with us.
+That'll be a major focus of ours for the year ahead.
+
+### What's the statement that most defines your migration experience?
+
+<img src="/static/blog/2021-survey/24.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="It was tricky but MUI's documentation and resources helped me to get it right: 43%; It was smooth and it is now working fine: 38.5%; It wasn't good, I had many problems and it took me a long time to finish: 14.3%; It was bad and I ended up regretting it: 2.8%; Other: 1.4%" />
+
+<p class="blog-description">930 out of 1589 answered.</p>
+
+### What could MUI do to improve the migration experience?
+
+<img src="/static/blog/2021-survey/25.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="General documentation improvements: 43.2%; Automation: 20.7%: Fewer breaking changes: 15.6%; Fewer styling solution breaking changes: 8.5%; More tutorials: 5.4%; More styling solution migration tutorial: 4.4%; Migrate Core and X together: 1%" />
+
+<p class="blog-description">472 out of 1589 answered.</p>
+
+- **General documentation improvement:** Documentation plays a huge part in defining the developer experience.
+  We definitely need to refine our copywriting, provide more examples, and offer more beginner-friendly resources.
+- **Automation:** We prepared codemods to help with the v5 migration, given the number of changes.
+  This has been met with positive feedback, which is encouraging since it's our first time providing tools to automate the migration process.
```

Comment:
```suggestion
  This has been met with positive feedback, which is encouraging since it's our first time providing comprehensive tools to automate the migration process.
```

Reply:
We provided limited codemods previously.